openedx/core/djangoapps/content_tagging/rest_api/v1/views.py

```diff
@@ -143,12 +146,26 @@ def orgs(self, request, **_kwargs) -> Response:
 class ObjectTagOrgView(ObjectTagView):
     """
     View to create and retrieve ObjectTags for a provided Object ID (object_id).
-    This view extends the ObjectTagView to add Organization filters for the results.
+    This view extends the ObjectTagView to add Organization filters for the results,
+    and fires events when the tags are updated.
 
     Refer to ObjectTagView docstring for usage details.
     """
     filter_backends = [ObjectTagTaxonomyOrgFilterBackend]
 
+    def update(self, request, *args, **kwargs) -> Response:
+        """
+        Extend the update method to fire CONTENT_OBJECT_TAGS_CHANGED event
+        """
+        response = super().update(request, *args, **kwargs)
+        if response.status_code == 200:
+            object_id = kwargs.get('object_id')
+            CONTENT_OBJECT_TAGS_CHANGED.send_event(
+                time=now(),
+                content_object=ContentObjectData(object_id=object_id)
```

Comment:
Also I wonder, what's an `object_id`? What does it look like?

Reply:
The `object_id` is the id of the course or library block as a string that is being tagged. For example:

`block-v1:SampleTaxonomyOrg2+STC1+2023_1+type@vertical+block@f8de78f0897049ce997777a3a31b6ea0`

Reply:
Can we add that explanation to the event data docstring? [Here](https://github.com/openedx/openedx-events/pull/327/files#diff-ce3d23f91ef18fd93f59fd1cf0a2c5f88b98dfde3f3790cb22281e74edfad042R191) that would address all my comments. Thank you!

Reply:
Sure thing! Updated.